src/main.ts

```diff
@@ -88,16 +82,22 @@ async function startVue () {
     })
   }
 
+  // Initialize Keyloak Service
```

Comment:
I moved Sentry first, so it can capture any following errors.
I moved KC second, so that maybe it finishes initializing while HJ and LD init. (see 5970 for details)

Reply:
> I moved Sentry first, so it can capture all subsequent errors.
> I moved KC second, so that maybe it finishes initializing while HJ and LD init. (see 5970 for details)